site/source/docs/porting/connecting_cpp_and_javascript/embind.rst

```diff
@@ -199,6 +199,63 @@ For other code, that is not optimized by closure compiler, you do not need to
 make such changes. You also do not need it if you build without ``--closure 1``
 to enable the closure compiler.
 
+Class Fields
+================
+
+Exposing C++ fields to JavaScript as properties can be done either directly or
+through getters and setters. The various ways are shown below:
+
+.. code:: cpp
+
+    class SubClass {
+    public:
+      int i;
+    };
+
+    class MyClass {
+    public:
+      int x;
+      SubClass subClass;
```

Comment:
subclass is kind of a misnomer here, no?   InnerClass isn't much better.. NestedClass?   How about just `Foo`?